org.eclipse.winery.yaml.common/src/test/resources/builder/3_9_3_12-capability_types-1_1.yml

```diff
@@ -4,6 +4,7 @@ capability_types:
   DFEndpoint:
     derived_from: tosca.capabilities.Endpoint
     properties:
+    valid_source_types: [ Compute ]
```

Comment:
As far as I understood, these files are directly copied from the specification, aren't they?

I think, it is OK to add an additionally property. But then, please create a new .yml file and add a new test for that. I think, there is a Parameterized test for that, where files can easily be added.

Reply:
This yaml file relates to an example in the tosca specification. i see no reason, beside some religious believe, that these examples should not be extended in a valid way. anyways i hope the whole test suit gets refactored completely, as these examples do not serve (in no way!) as a complete test of the specification.

Reply:
@TStadelmaier has the task to write a test suite for the YAML parser. Any input for him is welcomed!